lib/internal/backend/repository/repository2.go

```diff
@@ -125,7 +125,9 @@ func (rb *RepositoryBackend) buildACIV2(layerIDs []string, dockerURL *types.Pars
 		return nil, nil, err
 	}
 	for _, closer := range closers {
```

Comment:
For ii, it's because I was to be very explicit that `layerFiles` must be the same length as `layerIDs` (which is clear in `layerFiles := make([]*os.File, len(layerIDs))`), and `closers` is just following the same pattern.